src/css/application.css

```diff
@@ -342,6 +342,31 @@ body {
   }
 }
 
+.instructions-editor {
+  height: 100%;
+  padding-top: calc(28px + 1rem);
+
+  & pre {
+    height: 100%;
+    margin: 0;
+    outline: 0;
+    word-wrap: break-word;
+    overflow-wrap: break-word;
+    white-space: normal;
+  }
+}
+
+.instructions-editor-menu {
```

Comment:
Suggest to follow BEM naming rules like the rest of the selectors.
`.instructions-editor-menu` to `.instructions-editor__menu`